modules/porous_flow/doc/content/modules/porous_flow/sqa/porous_flow_sdd.md

```diff
@@ -1 +1,65 @@
 !template load file=sqa/module_sdd.md.template category=porous_flow module=Porous Flow
+
+!template! item key=introduction
+The [!ac](MOOSE) {{module}} module is based on the MOOSE framework and thus inherits
+the unique features and base characteristics of the framework, as outlined in the [framework_sdd.md].
+Specific details unique to the module are outlined in this document.
+!template-end!
+
+!template! item key=system-scope
+!include porous_flow_srs.md start=system-scope-begin end=system-scope-finish
+!template-end!
+
+!template! item key=dependencies-and-limitations
+The {{module}} module inherits the
+[software dependencies and limitations of the MOOSE framework](framework_sdd.md#dependencies-and-limitations).
+The {{module}} module currently has the following limitations in functionality...
```

Comment:
I wasn't sure of the current set of limitations in the module, so I've left a spot for that here. Also please add any external software dependencies required for the module here.

Reply:
```suggestion
!template! item key=dependencies-and-limitations
The {{module}} module inherits the
[software dependencies and limitations of the MOOSE framework](framework_sdd.md#dependencies-and-limitations). It depends on the Fluid Properties module for all fluid property calculations, the Tensor Mechanics module for the mechanical aspects for coupled flow and geomechanics models, and the Geochemistry module for coupled flow and geochemical reactions.

The {{module}} module currently has the following limitations in functionality: As it depends on other physics modules for fluid properties, mechanics and geochemistry calculations, the {{module}} module is limited to the capability of each of the physics modules listed above. For example, the only fluids that can be used are those that are made available in the Fluid Properties module. Another limitation is that only a small selection of constitutive models for properties like relative permeability or porosity-permeability relationships are available to the user. If a user requires some other constitutive model, they must develop that functionality themselves. 

The {{module}} module was developed before the inclusion of Automatic Differentiation capability in MOOSE, and the physics kernels are not currently designed for AD materials, meaning that new functionality currently requires hand-coded Jacobian entries.
```

Reply:
I wasn't sure how to link to the other modules in the above. You might want to add links to them?

Reply:
Added links and some formatting edits.